packages/amplify-frontend-ios/index.js

```diff
@@ -59,22 +59,35 @@ async function executeAmplifyCommand(context) {
   await commandModule.run(context);
 }
 
-const postEvents = new Set(['PostInit', 'PostCodegenModels', 'PostPull']);
-
 async function handleAmplifyEvent(context, args) {
   const { frontend } = context.amplify.getProjectConfig();
   const isXcodeIntegrationEnabled = FeatureFlags.getBoolean('frontend-ios.enableXcodeIntegration');
   const isFrontendiOS = frontend === 'ios';
-  if (isFrontendiOS && isXcodeIntegrationEnabled && postEvents.has(args.event)) {
-    await amplifyApp.run({
-      skipEnvCheck: true,
-      platform: frontend,
-      skipInit: true,
-      internalOnlyIosCallback: true,
-    });
+  const isMacOs = process.platform === 'darwin';
+  if (!isFrontendiOS || !isXcodeIntegrationEnabled || !isMacOs) {
+    return;
+  }
+  context.print.info('Updating iOS project');
+  const projectPath = process.cwd();
```

Comment:
does this break if the command is run from a subdirectory of the project?

Reply:
Thanks for pointing that out, that's an `amplify-app` relic. It doesn't break but throws an error. 
I'll replace with `pathManager.findProjectRoot()` from `amplify-cli-core`